WhisperAPI/WhisperAPI/Services/SuggestionsService.cs

```diff
@@ -117,5 +117,10 @@ private bool IsElementValid(ISearchResultElement result)
 
             return true;
         }
+
+        private string WildCardToRegularExpression(string value)
+        {
+            return "^" + Regex.Escape(value).Replace("\\?", ".").Replace("\\*", ".*") + "$";
```

Comment:
You added support for question marks being a one character wild card, but you don't use it. If you don't use it, you should remove the support for this, it's just a bug source. We'll add support for this later if we need it.